File: WordSwell.ApiModels/TestCont/TestCallResultModel.cs
namespace WordSwell.ApiModels.TestCont;

/// <summary>
/// 테스트용 결과 모델
/// </summary>
public class TestCallResultModel
{
    /// <summary>
    /// 개체 테스트 1
    /// </summary>
    public TestCallModel? Call { get; set; }

    /// <summary>
    /// 개체 테스트 2
    /// </summary>
    public TestResultModel? Result { get; set; }
}

File: WordSwell.Frontend/src/Faculty/Backend/TestCont/TestCallResultModel.ts
import { TestCallModel } from '@/Faculty/Backend/TestCont/TestCallModel';
import { TestResultModel } from '@/Faculty/Backend/TestCont/TestResultModel';

/** 테스트용 결과 모델 */
export interface TestCallResultModel 
{
    /** 개체 테스트 1 */
    Call?: TestCallModel,
    /** 개체 테스트 2 */
    Result?: TestResultModel,
}